CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+* Sufficient testing occurred
+* Quality standards are upheld
+
+While it is certainly _possible_ that a quality pull request might be accepted right away, it should be expected that there will be some changes requested of the pull request, and issues that need to be discussed and resolved before it is approved. The pull request post itself is a great place to have these conversations, though contributors should not be afraid to directly message fellow developers either, when doing so is helpful to resolving any issues.
+
+### Implementing Changes
+
+#### Approval
+Once the review process is completed to the satisfaction of project enough project maintainers, a project maintainer will merge the pull request with the "master" branch, incorporating any changes within the pull request.
+
+At this point, the contributor's job is done! If your pull request made it this far, congratulations! Your contribution is now part of the project!
+
+#### Release Builds
+Periodically, the current lead developer(s) of Nucleus Coop will decide when enough changes have been incorporated into the master branch to justify a new release. After sufficient testing by project maintainers to ensure build stability, a release will be created and published alongside a change log of implemented changed as improvements. Releases include source code along with a more convenient installer and a "portable" (pre-compiled zip folder) option for users to run the program.
+
+Currently, there is no set release schedule, and releases will be published when ready and verified stable.
+
+### Code Standards
+* When in doubt, structure your code to err on the side of readability!
+* Comment large blocks of abstract code with concise descriptions.
+* Please write C# code following [standard conventions](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions). 
+
+### Tips and Troubleshooting
+
+#### Updating Submodules
+If you get an error about missing URL's when attempting to initalize and update the submodules, open up .gitmodules (in the project root folder) in a text editor and make sure that the following are listed.
+    
+Even if some of these dependencies are not in active use, the git "update" command may fail to execute if entries and URLs for **all submodule** dependencies cloned from your fork are not found in the .gitmodules file.
+
+    [submodule "Submodules/x360ce"]
+	    path = Submodules/x360ce
+	    url = https://github.com/lucasassislar/x360ce
+
+    [submodule "Submodules/ProtoInput"]
+        path = Submodules/ProtoInput
+        url = https://github.com/ilyaki/protoinput
+
+    [submodule "Submodules/nukeupdater"]
+        path = Submodules/nukeupdater
+        url = https://github.com/lucasassislar/nukeupdater
+        
+    [submodule "Submodules/TypescriptSyntaxPaste"]
+        path = Submodules/TypescriptSyntaxPaste
+        url = https://github.com/nhabuiduc/TypescriptSyntaxPaste.git

```

Comment:
This can be moved into a code block, like so: 

```ini
    [submodule "Submodules/x360ce"]
	    path = Submodules/x360ce
	    url = https://github.com/lucasassislar/x360ce

    [submodule "Submodules/ProtoInput"]
        path = Submodules/ProtoInput
        url = https://github.com/ilyaki/protoinput

    [submodule "Submodules/nukeupdater"]
        path = Submodules/nukeupdater
        url = https://github.com/lucasassislar/nukeupdater
        
    [submodule "Submodules/TypescriptSyntaxPaste"]
        path = Submodules/TypescriptSyntaxPaste
        url = https://github.com/nhabuiduc/TypescriptSyntaxPaste.git
```

Reply:
I fought with the .md formatting a lot with this part, lol. The current version on my fork actually displays similar to that, oddly enough (https://github.com/RangerTJ/splitscreenme-nucleus/blob/master/CONTRIBUTING.md). I'm not entirely sure _why_ though. It might be displaying as another block type and not a code block, since I didn't use the '>' lead in for it. 

I noticed the one on my fork has sharp edges vs. rounded edges on the block). I tried using a leading '>' earlier but it kept giving me really weird formatting issues, where the brackets would be outside the block but the indented code chunks would be contained within it (or depending on how I spaced things, it'd wrap the text vs. letting me indent/create new lines). I'll play with it some more and see if I can get it to behave, heh.

Reply:
Should now be implemented!